pkg/commands/profile/token.go

```diff
@@ -26,32 +29,42 @@ func NewTokenCommand(parent argparser.Registerer, g *global.Data) *TokenCommand
 	return &c
 }
 
+// By default tokens must be valid for at least 5 minutes to be
+// considered valid.
+const defaultTokenTTL int64 = 5 * 60
```

Comment:
The more golang idomatic thing to do here would be to store a `time.Duration`:

```suggestion
const defaultTokenTTL time.Duration = 5 * time.Minute
```

This also make its intention and value more clear to readers.

You can then change the signature of `checkTokenValidity()` to accept a `Duration`

Reply:
Thanks! Great suggestion.